code/iaas/service-discovery/server/src/main/java/io/cattle/platform/servicediscovery/process/InstancePurgePostListener.java

```diff
@@ -0,0 +1,63 @@
+package io.cattle.platform.servicediscovery.process;
```

Comment:
allocation happens on instance start for sure. But I do see instance releasing computing resource  are called during purge path and that is where I plugged disk space release in the purge path.